### PR TITLE
MOI 1.0

### PR DESCRIPTION
**Pull request by joaquimg**

### CODE REVIEW

**Comment by joaquimg** [comment]
@matbesancon waiting for approval to run :)

**Comment by odow** [comment]
677glk

**Comment by joaquimg** [comment]
Please :)

**Comment by matbesancon** [comment]
sorry about the delays! Am on the road for a bit

**Comment by odow** [comment]
You might say on the ROADef?

**Comment by matbesancon** [comment]
there are still minor things to adjust it seems, @joaquimg if you have the time

**Comment by matbesancon** [comment]
also @joaquimg you were invited as collaborator of the repo some time ago, if you accept you should be able to run CI directly :stuck_out_tongue:

**Comment by joaquimg** [comment]
Missed that! Just found in mailbox, but its expired. Can you re-send?
I will keep working on this!

**Comment by joaquimg** [comment]
Aaaand I am back asking for more CI, and a new collaborator invite XD.
All tests passing locally in win.

**Comment by odow** [comment]
I have CI power in the mean time

**Comment by joaquimg** [comment]
All fails are the same:
```julia
Dual Cone: Test Failed at /home/runner/work/MathOptSetDistances.jl/MathOptSetDistances.jl/test/projection_gradients.jl:215
  Expression: ≈(dΠ, grad_fdm1, atol = tol) || ≈(dΠ, grad_fdm2, atol = tol)
Stacktrace:
 [1] macro expansion
   @ /opt/hostedtoolcache/julia/nightly/x64/share/julia/stdlib/v1.9/Test/src/Test.jl:464 [inlined]
 [2] macro expansion
   @ ~/work/MathOptSetDistances.jl/MathOptSetDistances.jl/test/projection_gradients.jl:215 [inlined]
 [3] macro expansion
   @ /opt/hostedtoolcache/julia/nightly/x64/share/julia/stdlib/v1.9/Test/src/Test.jl:1357 [inlined]
 [4] macro expansion
   @ ~/work/MathOptSetDistances.jl/MathOptSetDistances.jl/test/projection_gradients.jl:210 [inlined]
 [5] macro expansion
   @ /opt/hostedtoolcache/julia/nightly/x64/share/julia/stdlib/v1.9/Test/src/Test.jl:1357 [inlined]
 [6] macro expansion
   @ ~/work/MathOptSetDistances.jl/MathOptSetDistances.jl/test/projection_gradients.jl:[175](https://github.com/matbesancon/MathOptSetDistances.jl/runs/5434678975?check_suite_focus=true#step:6:175) [inlined]
 [7] macro expansion
   @ /opt/hostedtoolcache/julia/nightly/x64/share/julia/stdlib/v1.9/Test/src/Test.jl:1357 [inlined]
 [8] top-level scope
   @ ~/work/MathOptSetDistances.jl/MathOptSetDistances.jl/test/projection_gradients.jl:32
Dual Cone: Test Failed at /home/runner/work/MathOptSetDistances.jl/MathOptSetDistances.jl/test/projection_gradients.jl:215
  Expression: ≈(dΠ, grad_fdm1, atol = tol) || ≈(dΠ, grad_fdm2, atol = tol)
Stacktrace:
 [1] macro expansion
   @ /opt/hostedtoolcache/julia/nightly/x64/share/julia/stdlib/v1.9/Test/src/Test.jl:464 [inlined]
 [2] macro expansion
   @ ~/work/MathOptSetDistances.jl/MathOptSetDistances.jl/test/projection_gradients.jl:215 [inlined]
 [3] macro expansion
   @ /opt/hostedtoolcache/julia/nightly/x64/share/julia/stdlib/v1.9/Test/src/Test.jl:1357 [inlined]
 [4] macro expansion
   @ ~/work/MathOptSetDistances.jl/MathOptSetDistances.jl/test/projection_gradients.jl:210 [inlined]
 [5] macro expansion
   @ /opt/hostedtoolcache/julia/nightly/x64/share/julia/stdlib/v1.9/Test/src/Test.jl:1357 [inlined]
 [6] macro expansion
   @ ~/work/MathOptSetDistances.jl/MathOptSetDistances.jl/test/projection_gradients.jl:175 [inlined]
 [7] macro expansion
   @ /opt/hostedtoolcache/julia/nightly/x64/share/julia/stdlib/v1.9/Test/src/Test.jl:1357 [inlined]
 [8] top-level scope
   @ ~/work/MathOptSetDistances.jl/MathOptSetDistances.jl/test/projection_gradients.jl:32
```

will give some tries

**Comment by joaquimg** [comment]
Thanks @odow , I havent noticed that.
Since 1.6 is doing well I switched to its rng.

**Comment by joaquimg** [comment]
last commit should fix #63

**Comment by joaquimg** [comment]
All Passing!
Good to merge, for me.

File: test/Project.toml
Comment (matbesancon) [inline test/Project.toml:15]:
```suggestion
SCS = "=1.1.0"

```

**Review by matbesancon — APPROVED** [review]

**Comment by matbesancon** [comment]
LGTM! @joaquimg feel free to merge when this passes CI

**Pull request merged**
